db/migrate/20210621161519_add_applying_for_bursary_to_traine.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddApplyingForBursaryToTraine < ActiveRecord::Migration[6.1]
+  def change
+    add_column :trainees, :applying_for_bursary, :boolean
```

Comment:
The problem with doing that is that it would be hard to then default the question on the form to being unselected. We want the users to deliberately select if the trainee is applying for a bursary or not, instead of just clicking through.

Reply:
Discussed offline